tests/format/flow/no-semi/flow-class-properties.js

```diff
@@ -0,0 +1,7 @@
+class A {
+  +one = val();
+  +two = val();
```

Comment:
Let's add one computed property test, and change one of them to `-`.

Reply:
And better add one property that is a function

```js
+one = function () {}
```

Reply:
Added these in 60230f9b3

Reply:
I also added the `-` variance sigil and function property to the changelog. Didn't add computed property there because that wasn't changed in this PR.